packages/mdc-ripple/_mixins.scss

```diff
@@ -389,14 +423,36 @@
   $color,
   $has-nested-focusable-element,
   $opacity-modifier: 0,
-  $query: mdc-feature-all()
+  $query: mdc-feature-all(),
+  $ripple-target-opt: null,
 ) {
-  @include mdc-states-base-color($color, $query);
-  @include mdc-states-hover-opacity(mdc-states-opacity($color, hover) + $opacity-modifier, $query);
+  @include mdc-ripple-target-selector($ripple-target-opt) {
+    @include mdc-states-base-color($color, $query);
+  }
+
+  @include mdc-states-hover-opacity(
+    mdc-states-opacity($color, hover) + $opacity-modifier,
+    $query,
+    $ripple-target-opt);
```

Comment:
We need to name $query and $ripple-target for future proofing. 

```scss
@include mdc-states-hover-opacity(
  mdc-states-opacity($color, hover) + $opacity-modifier,
    $query: $query,
    $ripple-target: $ripple-target);
```

This way the argument order doesn't matter.

Same for across this file.